### PR TITLE
Multidimensional content widget + IPC guidance - content duplication

### PR DESCRIPTION
**Pull request by mick-prysm**
Previews:

 - Quickstart: https://deploy-preview-549--prysmdocs.netlify.app/docs/install/install-with-script
 - Merge prep: https://deploy-preview-549--prysmdocs.netlify.app/docs/prepare-for-merge
 - JWT: https://deploy-preview-549--prysmdocs.netlify.app/docs/execution-node/authentication

### CODE REVIEW

File: website/docs/prepare-for-merge.md [hunk 1]
```diff
@@ -51,18 +54,58 @@ Keep the following checklist in mind:
...
+
+<br />
+
+<Tabs className="with-label hidden-in-jwt-guide" groupId="protocol" default="jwt" values={[
+        {label: 'IPC', value: 'ipc'},
+        {label: 'HTTP-JWT', value: 'jwt'},
+    ]}>
+    <TabItem value="ipc">
+
+<h2>Configure execution node</h2>
```
Comment (mick-prysm) [inline website/docs/prepare-for-merge.md:73]: The indents here are strange but required due to JSX / Docusaurus :(

File: website/src/components/MultiDimensionalContentWidget.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+			if (jwtOnly()) {
+				selectByText('HTTP-JWT');
+				disableByText('IPC');
+			}
+
+			tabElements.forEach(element => {
+				var isLabel = element.textContent.indexOf(":") > -1;
+
+				if (isLabel) {
+					// unbind so tab can't be selected
```
Comment (mick-prysm) [inline website/src/components/MultiDimensionalContentWidget.js:72]: this unbinds the event handlers that docusaurus uses to handle tabs, allowing us repurpose their tabs as option labels

File: website/src/components/MultiDimensionalContentWidget.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+					element.outerHTML = element.outerHTML;
+				} else {
+					element.addEventListener("click", function (event) {
+						var targetElement = event.target;
+						var textContent = targetElement.textContent;
+
+						if (textContent == 'Besu' || textContent == 'Nethermind') {
```
Comment (mick-prysm) [inline website/src/components/MultiDimensionalContentWidget.js:79]: this is a little janky 

 - if another developer `selected` the wrong item, they could enter an infinite loop
 - lots of magic strings 
 
IMO this is good enough for now barring any bugs, I can improve the devex and maintainability as part of a debt payoff PR

File: website/src/components/MultiDimensionalContentWidget.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+								disableByText('Nethermind');
+							}
+						} else if (textContent == 'HTTP-JWT') {
+							enableByText('Besu');
+							enableByText('Nethermind');
+						}
+					}, false)
+				}
+			});
+			console.log("start with the user, work backwards to the technology...");
```
Comment (mick-prysm) [inline website/src/components/MultiDimensionalContentWidget.js:106]: I can remove this if anyone is opposed, my own little personal touch / easter egg

File: website/src/css/custom.css [hunk 5]
```diff
@@ -1,387 +1,401 @@
 :root {
-    --prysm-pink: #c72a7d;
+  --prysm-pink: #c72a7d;
```
Comment (mick-prysm) [inline website/src/css/custom.css:2]: CSS is now generated via LESS (see below)

File: website/src/css/custom.less [hunk 6]
```diff
@@ -0,0 +1,521 @@
+:root {
```
Comment (mick-prysm) [inline website/src/css/custom.less:1]: Using LESS will allow us to build much more maintainable styling for our docs

File: website/src/css/custom.less [hunk 7]
```diff
@@ -0,0 +1,521 @@
...
+    padding: 10px !important;
+    display: none !important;
+    /* can remove display style if we want to reintroduce */
+}
+
+footer>.col {
+    text-align: right;
+}
+
+footer>.col a,
```
Comment (mick-prysm) [inline website/src/css/custom.less:267]: these can all be grouped and organized using LESS conventions in a separate PR - this CSS is old CSS pulled into the new LESS file

File: website/docs/install/partials/_quickstart-intro.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+:::info Already running a node?
+
+This guidance is targeted at new users. If you're already running a node, see [Prepare for The Merge](../prepare-for-merge.md).
+
+:::
+
+<br />
+
+First, <strong>select a quickstart configuration</strong>. If you're looking for the simplest configuration, select `Geth` and `IPC`:
```
Comment (mick-prysm) [inline website/docs/install/partials/_quickstart-intro.md:12]: cc @nisdas (I'll @ you for the IPC bits to make your review a little easier)

File: website/docs/install/partials/_quickstart-run-beacon-node.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+    ]}>
+      <TabItem value="mainnet">  
+        <p class='hidden-in-jwt-guide'>Use the following command to start a beacon node that connects to your local execution node:</p>
+        <Tabs groupId="protocol" defaultValue="jwt" values={[
+            {label: 'JWT', value: 'jwt'},
+            {label: 'IPC', value: 'ipc'}
+            ]}>
+                <TabItem value="jwt"><pre><code>prysm.bat beacon-chain --execution-endpoint=http://localhost:8551 --jwt-secret=path/to/jwt.hex --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9</code></pre></TabItem>
+                <TabItem value="ipc">
+                  <div class="admonition admonition-info alert alert--info"><div class="admonition-content"><p><code>--http-web3provider</code> is deprecated and has been replaced with <code>--execution-endpoint</code>, but IPC currently only works through <code>--http-web3provider</code> on Windows. This will be fixed in our next release. You can safely ignore any related "deprecated flag" warnings you see in the meantime.</p></div></div>
```
Comment (mick-prysm) [inline website/docs/install/partials/_quickstart-run-beacon-node.md:25]: #ipc

File: website/docs/install/partials/_quickstart-run-beacon-node.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+        <Tabs groupId="protocol" defaultValue="jwt" values={[
+            {label: 'JWT', value: 'jwt'},
+            {label: 'IPC', value: 'ipc'}
+            ]}>
+                <TabItem value="jwt"><pre><code>prysm.bat beacon-chain --execution-endpoint=http://localhost:8551 --prater --jwt-secret=path/to/jwt.hex --genesis-state=genesis.ssz --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9</code></pre></TabItem>
+                <TabItem value="ipc">
+                <div class="admonition admonition-info alert alert--info"><div class="admonition-content"><p><code>--http-web3provider</code> is deprecated and has been replaced with <code>--execution-endpoint</code>, but IPC currently only works through <code>--http-web3provider</code> on Windows. This will be fixed in our next release. You can safely ignore any related "deprecated flag" warnings you see in the meantime.</p></div></div>
+                <pre><code>prysm.bat beacon-chain --http-web3provider=//./pipe/geth.ipc --prater --genesis-state=genesis.ssz --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9</code></pre></TabItem>
```
Comment (mick-prysm) [inline website/docs/install/partials/_quickstart-run-beacon-node.md:39]: #ipc

File: website/docs/install/partials/_quickstart-run-beacon-node.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+        {label: 'Ropsten', value: 'ropsten'}
+    ]}>
+      <TabItem value="mainnet">
+        <p>Use the following command to start a beacon node that connects to your local execution node:</p>
+        <Tabs groupId="protocol" defaultValue="jwt" values={[
+            {label: 'JWT', value: 'jwt'},
+            {label: 'IPC', value: 'ipc'}
+            ]}>
+                <TabItem value="jwt"><pre><code>./prysm.sh beacon-chain --execution-endpoint=http://localhost:8551 --jwt-secret=path/to/jwt.hex --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9</code></pre></TabItem>
+                <TabItem value="ipc"><pre><code>./prysm.sh beacon-chain --execution-endpoint=$HOME/.ethereum/geth.ipc --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9</code></pre></TabItem>
```
Comment (mick-prysm) [inline website/docs/install/partials/_quickstart-run-beacon-node.md:82]: #ipc (macos / linux)

File: website/docs/install/partials/_quickstart-run-execution-node.md [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+        {label: 'Sepolia', value: 'sepolia'},
+        {label: 'Ropsten', value: 'ropsten'}
+    ]}>
+      <TabItem value="mainnet">
+        <Tabs className='tabs-hidden-in-jwt-guide' groupId="protocol" defaultValue="jwt" values={[
+            {label: 'JWT', value: 'jwt'},
+            {label: 'IPC', value: 'ipc'}
+            ]}>
+                <TabItem value="jwt"><pre><code>Nethermind.Runner --JsonRpc.Enabled true --HealthChecks.Enabled true --HealthChecks.UIEnabled true --JsonRpc.JwtSecretFile=../consensus/jwt.hex --JsonRpc.Host=0.0.0.0</code></pre></TabItem>
+                <TabItem value="ipc"><div class="admonition admonition-info alert alert--info"><div class="admonition-content"><p>Content under construction.</p></div></div></TabItem>
```
Comment (mick-prysm) [inline website/docs/install/partials/_quickstart-run-execution-node.md:23]: These "under construction" bits are effectively "unsupported at the moment" and shouldn't ever be displayed - the react component auto-hides these

File: website/docs/install/partials/_quickstart-intro.md (hunk 8 above)
Comment (nisdas) [reply website/docs/install/partials/_quickstart-intro.md:12]: I think nethermind supports IPC too ? Did you ever manage to get IPC working on nethermind

**Review by nisdas — COMMENTED** [review]
In the quickstart option, it seems like a lot of the guidance defaults to nethermind( even if chose the geth toggle)

File: website/docs/install/partials/_quickstart-intro.md (hunk 8 above)
Comment (mick-prysm) [reply website/docs/install/partials/_quickstart-intro.md:12]: I'll unlock this path in a separate PR :)

**Pull request merged**